doc/whats_new/v0.21.rst

```diff
@@ -40,6 +40,14 @@ Support for Python 3.4 and below has been officially dropped.
 - An entry goes here
 - An entry goes here
 
+:mod:`sklearn.cluster`
```

Comment:
This is in master, and I saw there was a conflict for this file. Hopefully this resolves it.

Reply:
I think this is a bad fix to the conflict. Please just `git checkout master doc/whats_new/v0.21.rst`

Reply:
Yeah, this diff isn't right. I should say: `git checkout iterativeimputer doc/whats_new/v0.21.rst`